libs/schemas/src/entities/user.schemas.ts

```diff
@@ -59,6 +59,16 @@ export const Address = z.object({
   updated_at: z.any(),
 });
 
+export const SsoToken = z.object({
+  id: PG_INT.optional(),
```

Comment:
@jnaviask can we have multiple SSO tokens for any given address id? This surrogate id can be removed and we can use the address_id as PK if that's the case.

Reply:
@Rotorsoft A given address_id should always only have one or (in most cases) no SsoTokens.